src/test/java/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,125 @@
+package hw2.ex1;
+
+import hw2.AbstractBaseSeleniumTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static hw2.ex1.HomePageDataProvider.benefitIcons;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomePageTest extends AbstractBaseSeleniumTest {
+
+    @Test(dataProviderClass = HomePageDataProvider.class, dataProvider = "MenuElements")
+    public void headerMenuTest(String[] menu) {
+        //6. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> headerMenu = driver.findElements(By.cssSelector(".uui-navigation.nav.navbar-nav.m-l8>li"));
+        assertEquals(headerMenu.size(), 4);
+
+        for (int i = 0; i < 4; i++) {
+            assertEquals(headerMenu.get(i).getText(), menu[i]);
+            assertTrue(headerMenu.get(i).isDisplayed());
+        }
+    }
+
+    @Test
+    public void benefitImagesDisplayedTest() {
```

Comment:
How you define the order of the tests?

Reply:
Most tests depend on Login and open page only. That's why I decided the order doesn't matter. For tests with dependencies there are attributes "dependOnMethods" and "priority"